### PR TITLE
Tests/ethereum

### PR DESCRIPTION
**Pull request by FlorianHuc**
add workflow to run the ethereum tests

This PR implements issue(s) #1013

### CODE REVIEW

File: .github/workflows/gradle-ethereum-tests.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        run: ./gradlew :reference-tests:referenceTests --tests "net.consensys.linea.generated.blockchain.BlockchainReferenceTest_583"
+
+      - name: Run general reference tests
+        run: ./gradlew referenceGeneralStateTests
+        env:
+          JAVA_OPTS: -Dorg.gradle.daemon=false
+          CORSET_FLAGS: fields,expand,expand,expand
+
+      - name: Run all reference tests
+        run: ./gradlew referenceTests
```
Comment (gauravahuja) [inline .github/workflows/gradle-ethereum-tests.yml]: .`/gradlew referenceTests` will run `referenceBlockchainTests` and  `referenceGeneralStateTests`  which makes the previous step `Run general reference tests` redundant.

So maybe remove the "Run all reference tests" and add a new step for `referenceBlockchainTests`

Comment (FlorianHuc) [reply .github/workflows/gradle-ethereum-tests.yml]: thanks i remove it. I am not adding block chain tests are they are not working right now.

**Comment by OlivierBBB** [comment]
Can we run the EVM test suite locally too ?

**Comment by FlorianHuc** [comment]
you can also run them locally. Girst you need to generate them: ./gradlew :reference-tests:generateGeneralStateReferenceTests

Then you can execute them: 
 ./gradlew referenceGeneralStateTests

**Comment by gauravahuja** [comment]
Currently `./gradlew :reference-tests:generateGeneralStateReferenceTests` generates tests with compilation issue. I just pushed a fix for that in this PR.

**Review by gauravahuja — APPROVED** [review]

**Pull request merged**